```
-----------------------------------------------------
Manifest: manifests\m\Microsoft\VC++2010Redist-x64\10.0.40219.325
-----------------------------------------------------
URI: https://www.microsoft.com/en-us/download/details.aspx?id=13523, Validation result: Failed, Http status code: ServiceUnavailable 
URI: https://download.microsoft.com/download/1/6/5/165255E7-1014-4D0A-B094-B6A430A6BFFC/vcredist_x64.exe, Validation result: Passed

****************************************************************************************
Manifest Content Validation
****************************************************************************************
Status: Completed
##[error] Result: Failed
```

--- a/manifests/m/Microsoft/VC++2010Redist-x64/10.0.40219.325/Microsoft.VC++2010Redist-x64.installer.yaml
+++ b/manifests/m/Microsoft/VC++2010Redist-x64/10.0.40219.325/Microsoft.VC++2010Redist-x64.installer.yaml
@@ -1,5 +1,3 @@
-# Created using wingetcreate
-
 PackageIdentifier: Microsoft.VC++2010Redist-x64
 PackageVersion: 10.0.40219.325
 Installers:

--- a/manifests/m/Microsoft/VC++2010Redist-x64/10.0.40219.325/Microsoft.VC++2010Redist-x64.locale.en-US.yaml
+++ b/manifests/m/Microsoft/VC++2010Redist-x64/10.0.40219.325/Microsoft.VC++2010Redist-x64.locale.en-US.yaml
@@ -1,5 +1,3 @@
-# Created using wingetcreate
-
 PackageIdentifier: Microsoft.VC++2010Redist-x64
 PackageVersion: 10.0.40219.325
 PackageLocale: en-US

--- a/manifests/m/Microsoft/VC++2010Redist-x64/10.0.40219.325/Microsoft.VC++2010Redist-x64.locale.en-US.yaml
+++ b/manifests/m/Microsoft/VC++2010Redist-x64/10.0.40219.325/Microsoft.VC++2010Redist-x64.locale.en-US.yaml
@@ -2,8 +2,8 @@ PackageIdentifier: Microsoft.VC++2010Redist-x64
 PackageVersion: 10.0.40219.325
 PackageLocale: en-US
 Publisher: Microsoft Corporation
-PackageName: Microsoft Visual C++ 2010 Service Pack 1 Redistributable Package MFC Security Update (x64)
-PackageUrl: https://www.microsoft.com/en-us/download/details.aspx?id=13523
+PackageName: Microsoft Visual C++ 2010 x64 Redistributable
+PackageUrl: https://www.microsoft.com/en-us/download/details.aspx?id=26999
 License: Copyright (c) Microsoft Corporation
 ShortDescription: The Microsoft Visual C++ 2010 SP1 Redistributable Package (x64) installs runtime components of Visual C++ Libraries required to run 64-bit applications developed with Visual C++ 2010 SP1 on a computer that does not have Visual C++ 2010 SP1 installed.
 Tags:

--- a/manifests/m/Microsoft/VC++2010Redist-x64/10.0.40219.325/Microsoft.VC++2010Redist-x64.yaml
+++ b/manifests/m/Microsoft/VC++2010Redist-x64/10.0.40219.325/Microsoft.VC++2010Redist-x64.yaml
@@ -1,5 +1,3 @@
-# Created using wingetcreate
-
 PackageIdentifier: Microsoft.VC++2010Redist-x64
 PackageVersion: 10.0.40219.325
 DefaultLocale: en-US

